pkg/state/file_db.go

```diff
@@ -18,26 +19,21 @@ const (
 	TypeFile DBType = iota
```

Comment:
let's remove the field `DBType`

Reply:
the FileName state.json is confusing, because it still gets created when running the tests locally, not sure where it persist in the production build, if we still plan to use it its behavior needs to be documented